documentation/pom.xml

```diff
@@ -210,7 +210,6 @@
                                         <pdf-style>hibernate</pdf-style>
                                         <pdf-fontsdir>${asciidoctor.aggregated-resources-dir}/theme/fonts</pdf-fontsdir>
                                         <pagenums/>
-                                        <idprefix/>
```

Comment:
I think this change will break all the unqualified anchors which doesn't sound as a good idea.

Is there a reason why you removed it? If there is an issue with Asciidoctor, we should open a bug there.

Reply:
See the discussions I linked to.. also, this is only PDF so it's not going to break any links.

Reply:
Ah, yes sorry, I didn't have the full context :).